### DISCUSSION TITLE
speaker information missing in Json file obtained through code.

### DISCUSSION THREAD

**Discussion by Parth-Lightbulb** [deepgram/community#392] [General help]
I tried extracting transcript Json using API key.
didn't get the speaker information there.
then used 
curl \
  --request POST \
  --header 'Authorization: Token YOUR_DEEPGRAM_API_KEY' \
  --header 'Content-Type: audio/wav' \
  --data-binary @youraudio.wav \
  --url 'https://api.deepgram.com/v1/listen?diarize=true'
  
  made a right code but not getting any result.
  the error I am getting is 
  
 ConnectionError: ('Connection aborted.', ConnectionResetError(10054, 'An existing connection was forcibly closed by the remote host', None, 10054, None))
 
  after a while got this
  
 ConnectionError: HTTPConnectionPool(host='api.deepgram.com', port=80): Max retries exceeded with url: /v1/listen?diarize=true&diarize=True (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000021E1436D0D0>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))
 
 Any suggestions?

**Comment by jjmaldonis** [comment]
Hi @Parth-Lightbulb, your curl request is correct. Have you tried with a different audio file? If a different audio file works, can you tell us more about the audio file you're submitting? The output of `ffprobe youraudio.wav` would be very helpful.

**Reply by Parth-Lightbulb** [reply]
right now, after trying with different audio file I am getting this series errors.

TimeoutError: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond

During handling of the above exception, another exception occurred:


NewConnectionError: <urllib3.connection.HTTPConnection object at 0x000001B8AE40EB50>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond

During handling of the above exception, another exception occurred:


MaxRetryError: HTTPConnectionPool(host='api.deepgram.com', port=80): Max retries exceeded with url: /v1/listen?diarize=true&diarize=True (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001B8AE40EB50>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))

During handling of the above exception, another exception occurred:



ConnectionError: HTTPConnectionPool(host='api.deepgram.com', port=80): Max retries exceeded with url: /v1/listen?diarize=true&diarize=True (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001B8AE40EB50>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond')) 


And below I am providing you the data you asked about the audio file.

C:\ffmpeg\bin>ffprobe dis_output.wav
ffprobe version 2023-02-16-git-aeceefa622-full_build-www.gyan.dev Copyright (c) 2007-2023 the FFmpeg developers
  built with gcc 12.2.0 (Rev10, Built by MSYS2 project)
  configuration: --enable-gpl --enable-version3 --enable-static --disable-w32threads --disable-autodetect --enable-fontconfig --enable-iconv --enable-gnutls --enable-libxml2 --enable-gmp --enable-bzlib --enable-lzma --enable-libsnappy --enable-zlib --enable-librist --enable-libsrt --enable-libssh --enable-libzmq --enable-avisynth --enable-libbluray --enable-libcaca --enable-sdl2 --enable-libaribb24 --enable-libdav1d --enable-libdavs2 --enable-libuavs3d --enable-libzvbi --enable-librav1e --enable-libsvtav1 --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxavs2 --enable-libxvid --enable-libaom --enable-libjxl --enable-libopenjpeg --enable-libvpx --enable-mediafoundation --enable-libass --enable-frei0r --enable-libfreetype --enable-libfribidi --enable-liblensfun --enable-libvidstab --enable-libvmaf --enable-libzimg --enable-amf --enable-cuda-llvm --enable-cuvid --enable-ffnvcodec --enable-nvdec --enable-nvenc --enable-d3d11va --enable-dxva2 --enable-libvpl --enable-libshaderc --enable-vulkan --enable-libplacebo --enable-opencl --enable-libcdio --enable-libgme --enable-libmodplug --enable-libopenmpt --enable-libopencore-amrwb --enable-libmp3lame --enable-libshine --enable-libtheora --enable-libtwolame --enable-libvo-amrwbenc --enable-libilbc --enable-libgsm --enable-libopencore-amrnb --enable-libopus --enable-libspeex --enable-libvorbis --enable-ladspa --enable-libbs2b --enable-libflite --enable-libmysofa --enable-librubberband --enable-libsoxr --enable-chromaprint
  libavutil      58.  1.100 / 58.  1.100
  libavcodec     60.  2.100 / 60.  2.100
  libavformat    60.  2.100 / 60.  2.100
  libavdevice    60.  0.100 / 60.  0.100
  libavfilter     9.  2.100 /  9.  2.100
  libswscale      7.  0.100 /  7.  0.100
  libswresample   4.  9.100 /  4.  9.100
  libpostproc    57.  0.100 / 57.  0.100
Input #0, wav, from 'dis_output.wav':
  Metadata:
    encoder         : Lavf60.2.100
  Duration: 01:12:44.52, bitrate: 768 kb/s
  Stream #0:0: Audio: pcm_s16le ([1][0][0][0] / 0x0001), 48000 Hz, 1 channels, s16, 768 kb/s
